packages/AreaChart/src/AreaChart.tsx

```diff
@@ -173,11 +175,35 @@ const AreaChart: React.FunctionComponent<AreaChartProps> = (
 
   const [areaChartData, setAreaChartData] = React.useState<AreaChartData[]>();
 
+  const formatSpecialMonth = (date: DateTime) => {
+    if (date.locale !== 'en') {
+      return date.toFormat('MMM d');
```

Comment:
This should use .toLocaleString({
      month: 'short',
      day: 'numeric',
    });
    like it used to.